applications/OptimizationApplication/tests/filtering/explicit_filters_tests.py

```diff
@@ -0,0 +1,215 @@
+import numpy
+
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.OptimizationApplication as KratosOA
+from KratosMultiphysics.testing.utilities import ReadModelPart
+
+# Import KratosUnittest
+import KratosMultiphysics.KratosUnittest as kratos_unittest
+
+class TestExplicitVertexMorphingFilter(kratos_unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.model = Kratos.Model()
+        cls.model_part = cls.model.CreateModelPart("test")
+        cls.model_part.AddNodalSolutionStepVariable(Kratos.VELOCITY)
+        cls.model_part.AddNodalSolutionStepVariable(Kratos.ACCELERATION)
+        with kratos_unittest.WorkFolderScope(".", __file__):
+            ReadModelPart("solid", cls.model_part)
+
+        for node in cls.model_part.Nodes:
+            node.SetSolutionStepValue(Kratos.VELOCITY, Kratos.Array3([node.Id, node.Id + 1, node.Id + 2]))
+
```

Comment:
could you please add consistency checks, I mean make sure that for a uniform field and an integrated uniform field, we get uniform values. because these are critical checks for arbitrary non-unifom meshes. 
Say we a nodal/elemental uniform 1 field and then we filter it with arbitrary filter sizes, we need to get exactly 1 at each node/element. Also the same for an integrated uniform nodal/elemental field.

Reply:
Done :)